core/src/test/scala/org/apache/spark/SparkConfSuite.scala

```diff
@@ -241,6 +241,9 @@ class SparkConfSuite extends FunSuite with LocalSparkContext with ResetSystemPro
 
     conf.set("spark.yarn.applicationMaster.waitTries", "42")
     assert(conf.getTimeAsSeconds("spark.yarn.am.waitTime") === 420)
+
+    conf.set("spark.kryoserializer.buffer.max.mb", "1.1")
```

Comment:
I expect the test to fail, but my example code was bad (copy & pasted the wrong property). This should be `spark.kryoserializer.buffer.mb`. Sorry about that.